### PR TITLE
Addting RAT 3.0.0

### PR DESCRIPTION
**Pull request by pritamd47**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/rat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+  home: https://github.com/UW-SASWE/RAT
+  summary: Resevoir Monitoring using Satellite Remote Sensing
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - Pritam28877
```
Comment (dopplershift) [inline recipes/rat/meta.yaml:41]: Should this be:
```suggestion
    - pritamd47
```

File: recipes/rat/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    - rat
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/UW-SASWE/RAT
+  summary: Resevoir Monitoring using Satellite Remote Sensing
+  license: GPL-2.0-or-later
```
Comment (dopplershift) [inline recipes/rat/meta.yaml:36]:
```suggestion
  license: GPL-3.0-or-later
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor things to address.

**Review by pritamd47 — COMMENTED** [review]
Addressed changes and added another recipe maintainer

**Review by dopplershift — APPROVED** [review]
Recipe looks good (pending CI). I need confirmation from @SanchitMinocha that they are willing to be listed as a maintainer for this recipe.

**Comment by SanchitMinocha** [comment]
Thank you @pritamd47 for adding as a recipe maintainer. Yes, @dopplershift I am willing to be listed as a maintainer for this recipe.

**Pull request merged**